packages/store-ui/src/atoms/Button/UIButton.tsx

```diff
@@ -0,0 +1,24 @@
+import type { FC } from 'react'
+import React from 'react'
+import { Button } from 'theme-ui'
+
+import type { ButtonProps } from './Button'
+import RootButton from './Button'
+
+export type UIButtonProps = ButtonProps & {
+  variant: string
+}
+
+const UIButton: FC<UIButtonProps> = ({
+  children,
+  variant = 'sui-button',
+  ...props
+}) => {
+  return (
+    <Button as={RootButton} variant={variant} {...props}>
+      {children}
+    </Button>
+  )
+}
```

Comment:
Why this UIButton is needed, can use RootButton directly, don't? Variant props can be used like that 

```tsx
<button sx={{ variant }}>...</button>
```

Reply:
this component no more exists